pkg/kubelet/prober/manager.go

```diff
@@ -79,20 +79,34 @@ func NewManager(
 	runner kubecontainer.ContainerCommandRunner,
 	refManager *kubecontainer.RefManager,
 	recorder record.EventRecorder) Manager {
+	m := newManager(statusManager, livenessManager, runner, refManager, recorder)
+	m.start()
```

Comment:
Can we expose the start() method and have kubelet calls it. This makes it consistent with how we start other managers and avoid creating redundant goroutines during tests.